src/renderer/components/ParticipantsTile/AddParticipant.vue

```diff
@@ -0,0 +1,41 @@
+<template>
+  <form class="form" @submit.prevent="handleSubmit">
+    <b-field>
+      <b-input
+        placeholder="Type name"
+        type="text"
+        v-model="name"
+      />
+      <p class="control">
+        <b-input type="submit" value="Add" />
+      </p>
+    </b-field>
+  </form>
+</template>
+
+<script>
+export default {
+  props: {
+    onAdded: { type: Function },
```

Comment:
It's natural in React, but in Vue you should use events to handle child - parent communication. I'll continue below and in `ParticipantsTile/index.vue` file.
This should be removed.

docs: https://vuejs.org/v2/guide/components.html#Listening-to-Child-Components-Events

note: for custom events names you should always use kebab-case for event names - they won't be used used as variable / property names and they will be automatically transformed to lower case.